spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
It does not remove isolation. But by clearing the database it is easier to write tests that depend on state.

Furthermore, clearing the database takes time, at least 5 seconds per run. If you're running tests regularly, like you should, it adds up. Running tests in parallel also helps when you have hundreds of tests in future.